src/components/SellerPage/sample-merchant.ts

```diff
@@ -1,4 +1,4 @@
-import avatar from './images/female.svg'
+import avatar from './images/female.svg';
```

Comment:
The URL for the owner photo should come from `owner_image_url`

Reply:
that's sample data though, the whole thing should go. I was gonna work that out in a owner pic follow up where we standardize default pics.

Reply:
Sounds good!